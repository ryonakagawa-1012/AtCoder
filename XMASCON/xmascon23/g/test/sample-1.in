3
6 4 5 2 3 1
5 6 4 3 1 2
4 5 6 1 2 3
3 1 2 5 6 4
2 3 1 6 4 5
1 2 3 4 5 6
