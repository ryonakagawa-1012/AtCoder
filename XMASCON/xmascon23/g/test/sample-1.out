6
2 3 6 5 4 1
2 6 3 4 5 1
3 2 6 4 5 1
3 6 2 5 4 1
6 2 3 5 4 1
6 3 2 4 5 1
